packages/client/src/components/Card.tsx

```diff
@@ -22,45 +21,30 @@ function name([value, suit]: CardType) {
   return `${VALUES[value]} of ${SUIT[startCase(suit)]}`;
 }
 
-const suitList = Object.values(SUIT);
-function bgPosition([value, suit]: CardType) {
-  return `${(value - 1) * 11}% ${suitList.indexOf(suit) * 33}%`;
+function imgUrl([value, suit]: CardType) {
+  return require(`../images/${suit.toLowerCase()}/${padStart(`${value}`, 2, '0')}.jpg`).default;
 }
 
-const face = {
+const baseCard = {
   height: '13vw',
```

Comment:
Do we resize the assets here? Like, scaling down/up?

Reply:
we are not resizing, just fixing a height of the div seen on screen, the real image size won't change